### PR TITLE
documentation for tags addition to public apis

### PR DESCRIPTION
**Pull request by vimehta-zingbox**
## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: static/oas/iot/components/examples.yaml [hunk 1]
```diff
@@ -54,7 +61,40 @@ examples:
           risk_score: 0,
           risk_level: low,
           last_activity: 2018-05-31T18:39:37.404Z,
-          confidence_score: '90'
+          confidence_score: '90',
+          allTags: [
+                  {
+                  "tagType": "tagkey1",
```
Comment (Zhepu-zingbox) [inline static/oas/iot/components/examples.yaml:67]: let's use a meaningful value for `tagType`, `tagValue`

File: static/oas/iot/components/examples.yaml [hunk 2]
```diff
@@ -54,7 +61,40 @@ examples:
...
+  IoTTagListResponseExample:
+    value: { 
+      totalTags: 2,
+      tags: [
+        {
+          "_id": {
+            "tagId": "60f221a219e22f10003a965e"
+          },
+          "tagType": "tagkey1",
+          "tagValue": "tagval1",
```
Comment (Zhepu-zingbox) [inline static/oas/iot/components/examples.yaml:86]: let's use a meaningful value for `tagType`, `tagValue`
same as above

File: static/oas/iot/components/examples.yaml [hunk 3]
```diff
@@ -54,7 +61,40 @@ examples:
...
+          "tagId": "60f221a219e22f10003a965e",
+          "type": "custom"
+        },
+        {
+          "_id": {
+            "tagId": "60dad9446e89f01000b8950f"
+          },
+          "tagType": "Risk3",
+          "tagValue": "RiskHighFirst",
```
Comment (Zhepu-zingbox) [inline static/oas/iot/components/examples.yaml:95]: This is a good example. Let's use something like this to replace ones above

File: static/oas/iot/api/public/tagList.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
+parameters:
+- description: The customer ID specifies the API call for a specific tenant.
+  in: query
+  name: customerid
+  required: true
+  schema:
+    type: string
+- description: The tag typementions type of tag.
```
Comment (Zhepu-zingbox) [inline static/oas/iot/api/public/tagList.yaml:8]: `typementions` space needed

File: static/oas/iot/api/public/tagList.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  name: type
+  required: false  
+  schema:
+    type: string
+- description: This is the source of the user-defined tags assigned to IoT devices and must be followed by the string tenant.
+  in: query
+  name: source
+  required: true  
+  schema:
+    type: string
```
Comment (Zhepu-zingbox) [inline static/oas/iot/api/public/tagList.yaml:19]: Does this field have any default `enum` value?

**Review by Zhepu-zingbox — APPROVED** [review]

**Review by csestito — APPROVED** [review]

**Pull request merged**